### ISSUE TITLE
Wacom tablet x Chrome - scrolls without clicking

### ISSUE THREAD

**Issue by fishbucket** [noraesae/perfect-scrollbar#619]
This is a very specific scenario, but when using a Wacom tablet in Chrome, the scroll area moves as you hover over it, without clicking. Safari is fine, mouse is fine, trackpad is fine. It's just this Wacom x Chrome combination.

Any ideas? (Unfortunately I can't show a site to reproduce this, you need a wacom tablet and chrome to reproduce. But I'm seeing it on the main demo page here.)

**Comment by eelke** [comment]
I also ran into this behavior. Using Wacom Intuos 4 + Chrome 56.0.2924.87 (64-bit) on MacOS.

**Comment by eelke** [comment]
I noticed that while this behaviour appears on the Perfect Scrollbar homepage, it did not on this JSFiddle: https://jsfiddle.net/DanielApt/gbfLazpx/

The difference seems to be the `touch-action` on the `.ps-container` object. When it is set to `touch-action: auto;` (like on the homepage) we get this behaviour. When it is set to `touch-action: none;` it no longer happens. 

I don't know if changing this breaks anything else though. Maybe on (actual) touch devices?

**Comment by rianne-o** [comment]
I'm experiencing the same problem with my Wacom and Chrome. Setting the `touch-action` to `none`, doesn't make any difference in my case.
The example given in the JSFiddle however does work, but no due to the CSS.

It seems to be a difference between the versions. I used 0.6.12 when I was experiencing this issue and on an other site I had version 0.6.10 and all worked fine there. So I downgraded to 0.6.10 and the problem was gone. In 0.6.11 the problem occurs as well, so there has be something that has changed between version 0.6.10 and 0.6.11. 
Haven't looked further into the problem yet, but perhaps this is something to start with.
